ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -209,6 +209,9 @@ interface {{ ethernet_interface }}
 {%         if ethernet_interfaces[ethernet_interface].isis_network_point_to_point is arista.avd.defined(true) %}
    isis network point-to-point
 {%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].isis_circuit_type is arista.avd.defined %}
+   isis circuit-type {{ ethernet_interfaces[ethernet_interface].isis_circuit_type }}
```

Comment:
Are we rendering the config in the right order? With my test on 4.25.2F vEOS:
```POD1_LEAF01A(config-if-Et1)#sh ac
interface Ethernet1
   description P2P_LINK_TO_POD1_SPINE01_Ethernet1
   no shutdown
   mtu 9214
   no switchport
   ip address 192.168.16.9/31
   isis enable TEST
   isis circuit-type level-1
   isis metric 9999
   isis passive
   isis network point-to-point```

Reply:
Good catch. I will fix.